conductr_cli/conduct_unload.py

```diff
@@ -27,7 +27,9 @@ def unload(args):
         bundle_installation.wait_for_uninstallation(response_json['bundleId'], args)
 
     if not args.disable_instructions:
+        bundle_id = response_json['bundleId'] if args.long_ids else bundle_utils.short_id(response_json['bundleId'])
         log.info('Print ConductR info with: {} info{}'.format(args.command, args.cli_parameters))
+        log.info('Print bundle info with:   {} info{} {}'.format(args.command, args.cli_parameters, bundle_id))
```

Comment:
I don't think we should display the `conduct info <bundle id>` for bundle which has been unloaded.